### ISSUE TITLE
Get css links to work by linking them via assetic

### ISSUE THREAD

**Issue by skiadas** [skiadas/HanoverEvals#51]
See http://symfony.com/doc/current/cookbook/assetic/asset_management.html#controlling-the-url-used

Basically:

- The only thing that is visible to the user and their browser is the "app" folder. Nothing else.
- Stuff in the "web" folder is for assetic to manage and produce the appropriate corresponding files in "app" automatically when needed.
- So we need to either put our css/js stuff manually into the "app" folder, or use the web folder plus assetic. I think the latter would be the way to go. Any css/javascript that is not coming from a CDN should be managed by assetic.

**Comment by skiadas** [comment]
I was able to fix the link so that it seems our custom css. The path you provide it is meant to be the path starting at the "web" folder.

I moved the css folder from web/assets to sit just inside web instead.

**Comment by altermattw** [comment]
Yes, I saw that, too.  I'll change my stylesheet reference to go directly to /css instead of /assets/css

**Comment by skiadas** [comment]
I think I made the change already, you should just need to sync.

**Comment by altermattw** [comment]
Hi, Haris,

Sorry this is kind of clunky.  There was a minor conflict with a change I had made and thought I had committed with a “Close #51” comment, but apparently the commit didnt go through.  I’ll try the fork approach tonight.

On another note, I’m getting a login problem because there is no hanev_instructors table.  I checked the sqlite db and there is a hanev_courses_instructors table.  Any chance you’ve got some updated SQL syntax I could run to populate the DB?

—Bill



On Jun 22, 2016, at 11:45 AM, Haris Skiadas <notifications@github.com<mailto:notifications@github.com>> wrote:


I think I made the change already, you should just need to sync.

**Comment by skiadas** [comment]
Hm the table thing is weird. I didn't change anything that would affect that.

On all database problems at this stage the standard solution is three commands, run in this order:

- `php bin/console doctrine:schema:drop --force`
- `php bin/console doctrine:schema:create`
- `php bin/console doctrine:fixture:load`

**Comment by skiadas** [comment]
I just ran into the same issue on my end, will investigate.

**Comment by skiadas** [comment]
I think at least on my end the problem came from commit 724baf6272c21dfa0c5775528a96f8cd10c69551
where the config_dev file was changed to use sqlite. Since I was using mysql that caused the problems on my end. But it should not have affected you.

We should set you up with mysql, it's not too complex of an installation but it does require some care.

**Comment by altermattw** [comment]
Sorry about that. GitHub desktop was insisting that I commit all my changes before it would permit me to sync. I thought if a commit were problematic, you could delete it (and feel free to delete that one).

Bill

Sent from my iPhone

On Jun 22, 2016, at 12:07 PM, Haris Skiadas <notifications@github.com<mailto:notifications@github.com>> wrote:


I think at least on my end the problem came from commit 724baf6<https://github.com/skiadas/HanoverEvals/commit/724baf6272c21dfa0c5775528a96f8cd10c69551>
where the config_dev file was changed to use sqlite. Since I was using mysql that caused the problems on my end. But it should not have affected you.

We should set you up with mysql, it's not too complex of an installation but it does require some care.

-

**Comment by skiadas** [comment]
Well we can't really delete it per se once it's been pushed to github, but we can nullify its effect by overwriting it. It will keep being a problem though because then it would show up as a change on your end. It's probably simpler if I use sqlite for development, so that our systems agree.

Btw when GitHub doesn't let you sync like that, the solution I go with is:

- On the command line do "git stash". This hides the changes away.
- Do the sync.
- On the command line do "git stash apply". This brings the changes back. May result in some conflicts that you may need to resolve.

It's a pity stashes are not part of the desktop github app.

**Comment by skiadas** [comment]
This answer has some good info on the standard workflow:

http://stackoverflow.com/a/33069950

In particular the suggestion for using `git pull --rebase` is really good. Still assuming you have no uncommitted changes probably, but at least it will do the "rebase" step for you. This would be the command to use if you've made some local commits but haven't pushed them yet, and you want to pull in my recent updates from github, like you would do when you hit sync.

Basically, that sync button tries to do a bit too much if you ask me.
